src/app/public/modules/code-examples/code-example-theme.ts

```diff
@@ -0,0 +1,4 @@
+export enum SkyDocsCodeExampleTheme {
+  Default,
+  Modern
```

Comment:
If you used `string` enum, could you pass the value into the input as a string?
```
export enum SkyDocsCodeExampleTheme {
  Default = 'default',
  Modern = 'modern'
}
```
In the template...
```
<sky-docs-code-example theme="modern">
```

Reply:
Done.